spec/support/mock_herlper.rb

```diff
@@ -0,0 +1,37 @@
+module MockHelper
+  def album_search_request_success
+    stub_request(:get, 'https://jsonplaceholder.typicode.com/albums/1')
+      .to_return(
+        status: 200,
```

Comment:
I think you can use `:ok` to be a little more descriptive. https://ieftimov.com/post/how-rails-handles-status-codes/